modules/order/src/AvailabilityOrderProcessor.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace Drupal\commerce_order;
+
+use Drupal\commerce\AvailabilityManagerInterface;
+use Drupal\commerce_order\Entity\OrderInterface;
+
+/**
+ * Provides an order refresh processor for availability.
```

Comment:
"Provides an order processor that removes entities that are no longer available."?